src/Microsoft.ApplicationInsights.AspNetCore/Microsoft.ApplicationInsights.AspNetCore.xproj

```diff
@@ -4,21 +4,26 @@
     <VisualStudioVersion Condition="'$(VisualStudioVersion)' == ''">14.0</VisualStudioVersion>
     <VSToolsPath Condition="'$(VSToolsPath)' == ''">$(MSBuildExtensionsPath32)\Microsoft\VisualStudio\v$(VisualStudioVersion)</VSToolsPath>
   </PropertyGroup>
+  <Import Project="$(VSToolsPath)\DotNet\Microsoft.DotNet.Props" Condition="'$(VSToolsPath)' != ''" />
```

Comment:
Hmm I don't know. Bill Hiebert and Barry Tang would be more likely to know.

---
In reply to: [59234797](https://github.com/Microsoft/ApplicationInsights-aspnetcore/pull/142#discussion_r59234797) [](ancestors = 59234797)